src/app/auth/auth.js

```diff
@@ -0,0 +1,152 @@
+angular.module('prx.auth', ['prx.ui.nav'])
+.config(function (PRXDrawerProvider) {
+  PRXDrawerProvider.register({
```

Comment:
As mentioned above, here we register our sign in link, but it's not a link anymore.